### PR TITLE
Add Subject Tags and other missing tags

### PR DESCRIPTION
**Pull request by kafleg**
Add missing tags in TT2 theme

Trac ticket: https://core.trac.wordpress.org/ticket/58437

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-content/themes/twentytwentytwo/style.css [hunk 1]
```diff
@@ -158,4 +158,4 @@ body > .is-root-container > .wp-block-template-part > .wp-block-cover,
 
 .wp-block-post-comments ol.commentlist ul.children li:not(:last-child) {
 	margin-bottom: 1rem;
-}
+}
```
Comment (mukeshpanchal27) [inline src/wp-content/themes/twentytwentytwo/style.css]:
```suggestion
}

```

Add missing end line.

**Comment by SergeyBiryukov** [comment]
Hi there, thanks for the PR!

It appears that the PR includes some unrelated formatting changes removing the spaces around the `>` character. I think it makes those selectors a bit less readable and less consistent with other themes like Twenty Twenty-One, so it's probably out of scope for this PR. Could we just add the missing tags here?

**Comment by kafleg** [comment]
Hi @SergeyBiryukov 
That might because of the editor auto-formatting. And yes, we can simply add the tags here. 
How can I do that or you'll take care of it?

**Comment by ankitguptaindia** [comment]
> Hi @SergeyBiryukov That might because of the editor auto-formatting. And yes, we can simply add the tags here. How can I do that or you'll take care of it?

@kafleg 
I just created a new PR https://github.com/WordPress/wordpress-develop/pull/4532 in support of your PR. Copied the tags from your PR and added them in style.css. 
Please let me know if anything is missing there. Thanks for finding missing tags and creating a ticket.

**Comment by kafleg** [comment]
Looks good to me. 

Thank you

**Comment by mukeshpanchal27** [comment]
> Hi @SergeyBiryukov That might because of the editor auto-formatting. And yes, we can simply add the tags here. How can I do that or you'll take care of it?

Better to update your PR.

File: src/wp-content/themes/twentytwentytwo/style.css [hunk 2]
```diff
@@ -96,25 +96,25 @@ a:active {
  */
 
 .wp-site-blocks,
-body > .is-root-container,
+body>.is-root-container,
```
Comment (mukeshpanchal27) [inline src/wp-content/themes/twentytwentytwo/style.css]:
```suggestion
body > .is-root-container,
```

File: src/wp-content/themes/twentytwentytwo/style.css [hunk 3]
```diff
@@ -96,25 +96,25 @@ a:active {
...
 .edit-post-visual-editor__post-title-wrapper,
 .wp-block-group.alignfull,
 .wp-block-group.has-background,
 .wp-block-cover.alignfull,
-.is-root-container .wp-block[data-align="full"] > .wp-block-group,
-.is-root-container .wp-block[data-align="full"] > .wp-block-cover {
+.is-root-container .wp-block[data-align="full"]>.wp-block-group,
+.is-root-container .wp-block[data-align="full"]>.wp-block-cover {
```
Comment (mukeshpanchal27) [inline src/wp-content/themes/twentytwentytwo/style.css]:
```suggestion
.is-root-container .wp-block[data-align="full"] > .wp-block-group,
.is-root-container .wp-block[data-align="full"] > .wp-block-cover {
```

File: src/wp-content/themes/twentytwentytwo/style.css [hunk 4]
```diff
@@ -96,25 +96,25 @@ a:active {
...
-body > .is-root-container > .wp-block-cover,
-body > .is-root-container > .wp-block-template-part > .wp-block-group.has-background,
-body > .is-root-container > .wp-block-template-part > .wp-block-cover,
+.wp-site-blocks>.wp-block-group.has-background,
+.wp-site-blocks>.wp-block-cover,
+.wp-site-blocks>.wp-block-template-part>.wp-block-group.has-background,
+.wp-site-blocks>.wp-block-template-part>.wp-block-cover,
+body>.is-root-container>.wp-block-cover,
+body>.is-root-container>.wp-block-template-part>.wp-block-group.has-background,
+body>.is-root-container>.wp-block-template-part>.wp-block-cover,
```
Comment (mukeshpanchal27) [inline src/wp-content/themes/twentytwentytwo/style.css]:
```suggestion
.wp-site-blocks > .wp-block-group.has-background,
.wp-site-blocks > .wp-block-cover,
.wp-site-blocks > .wp-block-template-part>.wp-block-group.has-background,
.wp-site-blocks > .wp-block-template-part>.wp-block-cover,
body > .is-root-container > .wp-block-cover,
body > .is-root-container > .wp-block-template-part > .wp-block-group.has-background,
body > .is-root-container > .wp-block-template-part > .wp-block-cover,
```

File: src/wp-content/themes/twentytwentytwo/style.css [hunk 5]
```diff
@@ -126,7 +126,7 @@ body > .is-root-container > .wp-block-template-part > .wp-block-cover,
 .is-root-container .wp-block-columns .wp-block-column .wp-block[data-align="full"],
 /* We also want to avoid stacking negative margins. */
 .wp-site-blocks .alignfull:not(.wp-block-group) .alignfull,
-.is-root-container .wp-block[data-align="full"] > *:not(.wp-block-group) .wp-block[data-align="full"] {
+.is-root-container .wp-block[data-align="full"]>*:not(.wp-block-group) .wp-block[data-align="full"] {
```
Comment (mukeshpanchal27) [inline src/wp-content/themes/twentytwentytwo/style.css]:
```suggestion
.is-root-container .wp-block[data-align="full"] > *:not(.wp-block-group) .wp-block[data-align="full"] {
```

**Review by mukeshpanchal27 — COMMENTED** [review]
@kafleg Check some suggestions.

File: src/wp-content/themes/twentytwentytwo/style.css [hunk 6]
```diff
@@ -110,8 +110,8 @@ body > .is-root-container,
 .wp-site-blocks .alignfull,
 .wp-site-blocks > .wp-block-group.has-background,
 .wp-site-blocks > .wp-block-cover,
-.wp-site-blocks > .wp-block-template-part > .wp-block-group.has-background,
-.wp-site-blocks > .wp-block-template-part > .wp-block-cover,
+.wp-site-blocks > .wp-block-template-part>.wp-block-group.has-background,
+.wp-site-blocks > .wp-block-template-part>.wp-block-cover,
```
Comment (mukeshpanchal27) [inline src/wp-content/themes/twentytwentytwo/style.css]:
```suggestion
.wp-site-blocks > .wp-block-template-part > .wp-block-group.has-background,
.wp-site-blocks > .wp-block-template-part > .wp-block-cover,
```

**Review by mukeshpanchal27 — APPROVED** [review]
Thanks @kafleg for the changes. Look good to me.

**Comment by SergeyBiryukov** [comment]
Thanks for the PR! Merged in [r55965](https://core.trac.wordpress.org/changeset/55965).

**Pull request closed without merging**